lib/screens/RecipeScreen.dart

```diff
@@ -30,6 +34,148 @@ class _RecipePageState extends State<RecipePage> {
 
   @override
     Widget build(BuildContext context) {
-    return Scaffold();
+    return Scaffold(
+      appBar: AppBar(
+          title: const Text(
+            'SmartChef',
+            style: TextStyle(fontSize: 24, color: mainScheme),
+          ),
+          centerTitle: true,
+          backgroundColor: Colors.white,
+          leading: IconButton(
+            onPressed: () {},
+            icon: Icon(
+              Icons.search,
+              color: Colors.black,
+            ),
+            iconSize: 35,
```

Comment:
This is a different size because it is in a different location as oppose to other icons. This one is in the top appbar, whereas the others are in the bottom appbar. It wouldn't fit if it were bigger than it currently is.